src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerConnection.java

```diff
@@ -278,6 +278,25 @@ CallableStatement prepareCall(String sql, int nType, int nConcur, int nHold,
      */
     void setEnablePrepareOnFirstPreparedStatementCall(boolean value);
 
+    /**
+     * Returns the behavior for a specific connection instance. Returns one of the following:
+     * 1. prepxec, if sp_prepexec is set (default behavior)
+     * 2. prepare, if sp_prepare is set
+     *
+     * @return Returns current setting for prepareMethod connection property.
+     */
+    String getPrepareMethod();
+
+    /**
+     * Sets the behavior for the prepare method. Only the following string values are permitted:
+     * 1. prepexec, for use of sp_prepexec (default behavior)

```

Comment:
I don't think we should list this everywhere if we ever need to add a new option this would need to be updated in so many places. Should just reference a @link to PrepareMethod in SQLServerDriverPropertyInfo

Reply:
also pls fix spelling typo in line 304